### ISSUE TITLE
Running this stops my Virtual Desktop overlay function.

### ISSUE THREAD

**Issue by ianhawki** [vtjoeh/webex-desk-slide-presenter#1]
Running this code will prevent having my PC behind my image when using virtual backgrounds. Had to factory default to get it back to normal

Using CHANGE BACKGROUND and selecting PC Source will show you how it happens

**Comment by vtjoeh** [comment]
Thanks for bringing this up.  I'm sorry you needed to factory reset the device, sounds very frustrating.  There is a slightly simpler workaround I figured out. For your issue, I'm guessing the PC behind the image still worked, but it was stuck in the position last set by the macro versus full screen.  If anybody else gets this problem, the workaround fix at the moment is:
1) Disable the macro.   
2) Place a call. Let call connect.   
3)  On the Webex Desk screen, click "Share" --> Toggle On: "Show Me in Front of presentation" --> "Share in call"
4) Hang-up call.  

I'll try to make a fix or easier workaround in the next version (0.1.5.4) so no one else resets their system.

**Comment by ianhawki** [comment]
﻿Hi Joe,  I ended up added some code to the make in the reset area.

xCommand Cameras Background ForegroundParameters Reset

Seems address and reset the device back to normal

Regards
Ian Hawkins

**Comment by vtjoeh** [comment]
Thanks Ian, I added that command into the javascript code ver 0.1.5.4 and tested.  It works well.  The only thing to be aware of is when the PowerPoint presentation ends last virtual background except for HDMI/USB-C PC will show up.  However, once you are finished with the PowerPoint presentation, you should be able to select the PC Source as the virtual background from the touch panel.

**Comment by vtjoeh** [comment]
Closing the issue.  Fixed in version WebexDeskSlidePresenter.js ver 0.1.5.4.  Since I can't predict all combinations of clicks and connections that could leave the Webex Desk in an undesirable state, I also included the ability to reset the device by calling screenfix@example.com.  The call won't connect, but it does reset the camera and virtual background settings.  This could be accomplished by adding a button to the touchpanel/screen, but I did not want to clutter up the GUI.   @ianhawki  thanks for posting.
